src/main/java/net/glease/ggfab/ConfigurationHandler.java

```diff
@@ -30,7 +30,7 @@ private void setLanguageKeys() {
     }
 
     private void loadConfig() {
-        laserOCPenaltyFactor = config.getFloat("advasslinePenaltyFactor", "common.balancing", 0.25f, 0f, 10f, "Laser overclock penalty factor. This will incredibly change the game balance. Even a small step from 0.3 to 0.3 can have very significant impact. Tweak with caution!");
+        laserOCPenaltyFactor = config.getFloat("advasslinePenaltyFactor", "common.balancing", 0.3f, 0f, 10f, "Laser overclock penalty factor. This will incredibly change the game balance. Even a small step from 0.3 to 0.3 can have very significant impact. Tweak with caution!");
```

Comment:
What do you mean a change from 0.3 to 0.3 the same number can have a significant impact